backend/services/office_hours/ticket.py

```diff
@@ -272,6 +273,70 @@ def create_ticket(
                 "You cannot create multiple tickets at once."
             )
 
+        course_query = (
+            select(CourseSiteEntity)
+            .join(OfficeHoursEntity)
+            .where(OfficeHoursEntity.id == ticket.office_hours_id)
+        )
+        course_entity = self._session.scalars(course_query).one_or_none()
+
+        if course_entity is None:
+            raise CoursePermissionException(
+                "Cannot create a ticket for a course that does not exist."
+            )
+
+        # Check if tickets exceed maximum in 24hrs.
+        days_cutoff = datetime.now() + timedelta(days=-1)
```

Comment:
**question:** Is there a way to do this on a standard day rather than past 24 hours? Maybe basing the cutoff on the first OH event of the current day?